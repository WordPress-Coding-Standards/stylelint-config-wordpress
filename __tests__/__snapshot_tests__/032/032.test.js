'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '032', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '032' )],
		configFile: systemTestUtils.caseConfig( '032' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
