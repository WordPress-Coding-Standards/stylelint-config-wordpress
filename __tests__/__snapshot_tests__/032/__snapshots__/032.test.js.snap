// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`032 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "032/stylesheet.css",
    "warnings": Array [
      Object {
        "column": 461,
        "line": 4,
        "rule": "max-line-length",
        "severity": "error",
        "text": "Expected line length to be no more than 80 characters (max-line-length)",
      },
      Object {
        "column": 254,
        "line": 10,
        "rule": "max-line-length",
        "severity": "error",
        "text": "Expected line length to be no more than 80 characters (max-line-length)",
      },
      Object {
        "column": 86,
        "line": 14,
        "rule": "max-line-length",
        "severity": "error",
        "text": "Expected line length to be no more than 80 characters (max-line-length)",
      },
      Object {
        "column": 81,
        "line": 16,
        "rule": "max-line-length",
        "severity": "error",
        "text": "Expected line length to be no more than 80 characters (max-line-length)",
      },
    ],
  },
]
`;
